docs/get-started/get-started-with-wordpress-com.md

```diff
@@ -0,0 +1,21 @@
+---
+post_title: Get started with Wordpress.com
+---
+
+Wordpress.com offers managed hosting, which helps optimize your website for speed, security, and performance. Managed hosting takes care of many technical aspects of your site for you. WordPress.com has many plans with increasingly powerful features for different site types and budgets.
+
+Wordpress.com provides the tools, workflows, and infrastructure you need to build and deploy your website faster. Our platform supports importing existing Wordpress installations, and it has globally distributed, secure infrastucture for serving content from data centers near the user for optimal performance.
+
+While working on a site, Wordpress.com provides tools for collaboration on your project such as staging sites, coming soon mode, and real-time backups.
+
+## Before you begin
+
+To get started, create an account on Wordpress.com.
+
+### [Sign up](https://wordpress.com/start)
```

Comment:
I'd recommend not combining links and headings, outside of any internal anchor text. 

Headings are typically used for wayfinding, and it's best practice to link within the body text below. For instance, with the 'Sign up' part of the sentence.